setup.py

```diff
@@ -40,15 +40,13 @@
     'flake8-print >= 3.1.0, < 4.0.0',
     'flake8-rst-docstrings == 0.2.3',
     'mypy == 0.910',
-    'openfisca-country-template >= 3.10.0, < 4.0.0',
-    'openfisca-extension-template >= 1.2.0rc0, < 2.0.0',
```

Comment:
Have you tried:

```suggestion
     'openfisca-country-template --no-dependencies',
     'openfisca-extension-template --no-dependencies',
```

? 🙂

Reply:
If that does not work, I would find it useful to add a comment in this dependencies list explaining that those will be installed with `make`, in order to make them discoverable.

Reply:
It does not work. I'll leave it as a comment then,